### PR TITLE
feat: add a consolidated endpoint for current and prior sortitions

### PR DESCRIPTION
**Pull request by kantai**
### Description

This implements the lowest hanging fruit for https://github.com/stacks-network/stacks-core/issues/5200 by adding an endpoint that returns both the current and prior sortitions.

### CODE REVIEW

**Comment by wileyj** [comment]
```
error[E0560]: struct `SortitionsView` has no field named `latest_consensus_hash`
Error:   --> stacks-signer/src/tests/chainstate.rs:85:9
   |
85 |         latest_consensus_hash: cur_sortition.consensus_hash,
   |         ^^^^^^^^^^^^^^^^^^^^^ `SortitionsView` does not have this field
   |
   = note: all struct fields are already assigned

For more information about this error, try `rustc --explain E0560`.
error: could not compile `stacks-signer` (lib test) due to 1 previous error
```

we also need an openapi doc for the signer, would be nice to document this endpoint so it's not forgotten (i'll create an issue for that specifically though - doesn't seem appropriate to add to stacks-core's openapi doc)

**Review by obycode — DISMISSED** [review]
👍

**Review by obycode — APPROVED** [review]

File: stackslib/src/net/api/getsortition.rs [hunk 1]
```diff
@@ -302,7 +373,7 @@ impl HttpResponse for GetSortitionHandler {
         preamble: &HttpResponsePreamble,
         body: &[u8],
     ) -> Result<HttpResponsePayload, Error> {
-        let sortition_info: SortitionInfo = parse_json(preamble, body)?;
+        let sortition_info: Vec<SortitionInfo> = parse_json(preamble, body)?;
```
Comment (jcnelson) [inline stackslib/src/net/api/getsortition.rs:376]: Can you add test coverage to `stacksilb/src/net/api/tests/getsortition.rs`?

Comment (kantai) [reply stackslib/src/net/api/getsortition.rs:376]: Added in 5d860c60b7b2cdb6c69c9e1466028904d240394a

**Review by obycode — APPROVED** [review]
LGTM

**Review by jcnelson — APPROVED** [review]

**Pull request merged**